tests/unit/test_config.py

```
@@ -251,3 +251,106 @@ def test_api_finding_search_endpoint(self):
 		from secator.config import Config
 		config = Config.parse()
 		self.assertEqual(config.addons.api.finding_search_endpoint, 'findings/_search')
+
+
+@mock.patch('sys.stderr', devnull)
+class TestSecretFields(unittest.TestCase):
+
+	def test_secret_paths_contains_expected(self):
+		from pydantic import SecretStr
```

Flake8 (7.3.0)

[error] 'pydantic.SecretStr' imported but unused

(F401)